index.js

```diff
@@ -1,88 +1,134 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+var counter = 1
+var counter = 2
```

Comment:
This works, but we should get into the practice of using `let` when we want to reassign a variable. It will save you a lot of headache and heartache down the line! You could do something like..

```javascript
let counter = 1
counter = 2
```